code/modules/mob/living/simple_animal/guardian/types/standard.dm

```diff
@@ -2,6 +2,7 @@
 /mob/living/simple_animal/hostile/guardian/punch
 	melee_damage_lower = 20
 	melee_damage_upper = 20
+	next_move_modifier = 0.9 //attacks 20% faster
```

Comment:
Setting it to 0.5 results in doubled attack rate/speed so I'm pretty sure this is 20%